specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/sshPublicKeyExamples/SshPublicKey_GenerateKeyPair_EncryptionWithEd25519.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-09-01",
+    "sshPublicKeyName": "mySshPublicKeyName",
+    "parameters": {
+      "encryptionType": "RSA"
```

Comment:
This example is for ED25519, but i think the example is for RSA . content is probably switched ?